pkg/utils/test/load_test_data.go

```diff
@@ -11,6 +11,23 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+// LoadMasterConfig loads MasterConfig
+func LoadMasterConfig(file string) (*legacyconfigv1.MasterConfig, error) {
```

Comment:
@Danil-Grigorev has pointed to this piece. It looks OK to do, but @gildub is there other places to use LoadMasterConfig in the code?

Reply:
Yes definitely, that was my plan in following patch/PR to push it up.

Reply:
Awe! :)